FIAT/arnold_winther.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""Implementation of the Arnold-Winther finite elements."""
+
+# Copyright (C) 2016-2018 Lizao Li <lzlarryli@gmail.com>
+#
+# This file is part of FIAT.
+#
+# FIAT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# FIAT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with FIAT. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Actually, per #33 one should use an SPDX header line instead of this standard template...

Reply:
Fixed.